### DISCUSSION TITLE
`deno fmt --check` is too strict on md files

### DISCUSSION THREAD

**Discussion by erfanium** [denoland/deno#13298] [Ideas]
Markdown files are very handy so we can edit docs instantly via Github editor in seconds, without need to clone the repo and do editing stuff on local machine.

but current `deno fmt --check` behavior is too strict on markdown files. often I should do editing steps on my local machine to make sure there's no format mistakes. It's very time consuming.

`dprint-ignore-file` is not a solution, because we need have this behavior as default behavior to affect the community.

What do you think? my preferred solution is to just ignore .md files in `deno fmt --check`

**Comment by kitsonk** [comment]
Having different default inclusions between `deno fmt` and `deno fmt --check` is surprising to a user, and therefore I would not be supportive of it.

If `deno fmt --check` is a problem for you individually, then adding an `--ignore` value would be the solution for you without imposing the personal choice on the rest of the community.

**Reply by erfanium** [reply]
What about showing warnings instead of errors for md files?

It's not really about only my repos and I don't want keep it personal, that's why I made this discussion to decides about it together.
I'm just saying editing markdown documents should not require extra steps (cloning and formatting on local machine). that really helps community members to improve community documents without need to spend too much time.
Why not improve docs while we are surfing Github repos? keep that simple

**Reply by kitsonk** [reply]
Why are linting issues with markdown files any different than other linting issues?

**Reply by erfanium** [reply]
From this point of view, you are right. this is not such an idealistic solution. but I think the value of what we gain (easier doc improving) is greater than what we lose (losing `deno fmt --check` consistency by making md files different).
Also that was just a suggestion, better solutions are welcome.
